src/NUnitFramework/tests/nunit.framework.tests-sl-5.0.csproj

```diff
@@ -297,7 +297,6 @@
     <Compile Include="Syntax\TestCompiler.cs" />
     <Compile Include="Syntax\ThrowsTests.cs" />
     <Compile Include="Syntax\TypeConstraints.cs" />
-    <Compile Include="TestContextTests.cs" />
```

Comment:
Except for that, looks good.
On Oct 15, 2015 3:27 PM, "CharliePoole" <notifications@github.com> wrote:

> In src/NUnitFramework/tests/nunit.framework.tests-sl-5.0.csproj
> <https://github.com/nunit/nunit/pull/894#discussion_r42168878>:
>
> > @@ -297,7 +297,6 @@
> >      <Compile Include="Syntax\TestCompiler.cs" />
> >      <Compile Include="Syntax\ThrowsTests.cs" />
> >      <Compile Include="Syntax\TypeConstraints.cs" />
> > -    <Compile Include="TestContextTests.cs" />
>
> I'll put it back and merge if you think the rest looks OK.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/nunit/nunit/pull/894/files#r42168878>.
>